### ISSUE TITLE
CornerNet: Detecting Objects as Paired Keypoints

### ISSUE THREAD

**Issue by joyhuang9473** [number9473/nn-algorithm#299]
# CornerNet: Detecting Objects as Paired Keypoints #

- Author: Hei Law, Jia Deng
- Origin: https://arxiv.org/abs/1808.01244
- Related:
  - https://github.com/princeton-vl/CornerNet
    - [demo code](https://github.com/princeton-vl/CornerNet/pull/60)
    - [training on my own pictures](https://github.com/princeton-vl/CornerNet/issues/30)
    - [How to fine-tunning your pretrained cooco model?](https://github.com/princeton-vl/CornerNet/issues/23)
  - [CornerNet：目标检测算法新思路](https://zhuanlan.zhihu.com/p/41825737)
  - [Object Detection-CornerNet笔记](https://zhuanlan.zhihu.com/p/41759548)
  - http://pvl.cs.princeton.edu/

**Comment by joyhuang9473** [comment]
[39] Denet: Scalable real-time object detection with directed sparse sampling

**Comment by joyhuang9473** [comment]
Associative Embedding

[27] Associative embedding: End-to-end learning for joint detection and grouping

**Comment by joyhuang9473** [comment]
"pull" loss and "push" loss

[26] Pixels to graphs by associative embedding.

**Comment by joyhuang9473** [comment]
hourglass network

[28] Stacked hourglass networks for human pose estimation

**Comment by joyhuang9473** [comment]
networks similar to the hourglass network.

[35] Beyond skip connections: Top-down modulation for object detection.

[10] Dssd: Deconvolutional single shot detector.

[22] Feature pyramid networks for object detection

[19] Reverse connection with objectness prior networks for object detection

**Comment by joyhuang9473** [comment]
soft-nms [2]: Soft-nms: improving object detection with one line of code

- https://arxiv.org/abs/1704.04503
- https://github.com/bharatsingh430/soft-nms
